lib/active_model/serializer.rb

```diff
@@ -46,6 +47,10 @@ def self.inherited(base)
       super
     end
 
+    def self.type(type)
+      self._type = type
+    end
```

Comment:
I'm not sure it's narrow. Underscored class-level accessor will be accessible anyways.